build.zig

```diff
@@ -6,11 +6,50 @@ const Mode = std.builtin.Mode;
 const config = @import("./src/config.zig");
 const Shell = @import("./src/shell.zig");
 
+// TigerBeetle binary requires certain CPU feature and supports a closed set of CPUs. Here, we
+// specify exactly which features the binary needs. Client shared libraries might be more lax with
+// CPU features required.
+const supported_targets: []const CrossTarget = supported_targets: {
+    @setEvalBranchQuota(100_000);
+    var result: []const CrossTarget = &.{};
+    const tripples = .{
+        "aarch64-linux",
+        "aarch64-macos",
+        "x86_64-linux",
+        "x86_64-macos",
+        "x86_64-windows",
+    };
+    const cpus = .{
+        "baseline+aes+neon",
+        "baseline+aes+neon",
+        "x86_64_v3+aes",
+        "x86_64_v3+aes",
+        "x86_64_v3+aes",
+    };
+    for (tripples, cpus) |tripple, cpu| {
```

Comment:
```suggestion
    for (tuples, cpus) |tuple, cpu| {
```

Or `triples|triple`.

Reply:
Since `aarch64-linux` is more two parts, and also since `tuple` is more general: "a data structure consisting of multiple parts" so it doesn't matter if there are two or three.

Reply:
"triple" is a term of art here: https://www.flother.is/til/llvm-target-triple/

It can contain anywhere from two to five components, but definitely only a single "p"!

Reply:
Ah, thanks for the "triple" pointer!

Reply:
They missed the chance to call it a trippple. :disappointed: